vm/devices/storage/disk_nvme/nvme_driver/src/queue_pair.rs

```diff
@@ -65,11 +64,9 @@ const PER_QUEUE_PAGES_NO_BOUNCE_BUFFER: usize = 64;
 const SQ_ENTRIES_PER_PAGE: usize = PAGE_SIZE / SQ_ENTRY_SIZE;
 
 #[derive(Inspect)]
-pub(crate) struct QueuePair<T: AerHandler> {
+pub(crate) struct QueuePair<T: AerHandler, S: DeviceBacking> {
```

Comment:
nit (and this PR didn't start the concern): you can use descriptive single-letters. E.g.

`<A: AerHandler, D: DeviceBacking>`